### PR TITLE
operator portworx (1.10.1)

### PR DESCRIPTION
**Pull request by haibinxie**
Signed-off-by: CNBU Jenkins <cnbu-jenkins@purestorage.com>

Thanks submitting your Operator. Please check below list before you create your Pull Request.

### New Submissions

* [ ] Are you familiar with our [contribution guidelines](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-via-pr.md)?
* [ ] Have you [packaged and deployed](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md) your Operator for Operator Framework?
* [ ] Have you tested your Operator with all Custom Resource Definitions?
* [ ] Have you tested your Operator in all supported [installation modes](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#operator-metadata)?
* [ ] Have you considered whether you want use [semantic versioning order](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md#semver-mode)?
* [ ] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?
* [ ] Is operator [icon](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#operator-icon) set?

### Updates to existing Operators

* [x] Did you create a `ci.yaml` file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md)?
* [x] Is your new CSV pointing to the previous version with the `replaces` property if you chose `replaces-mode` via the `updateGraph` property in `ci.yaml`?
* [x] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#channels) defined in the `package.yaml` or `annotations.yaml` ?
* [x] Have you tested an update to your Operator when deployed via OLM?
* [x] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?

### Your submission should not

* [x] Modify more than one operator
* [x] Modify an Operator you don't own
* [x] Rename an operator - please remove and add with a different name instead
* [x] Modify any files outside the above mentioned folders
* [x] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [x] Description about the managed Application and where to find more information
2. [x] Features and capabilities of your Operator and how to use it
3. [x] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [x] Human readable name and 1-liner description about your Operator
* [x] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#categories)<sup>1</sup>
* [x] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [x] Links to the maintainer, source code and documentation
* [x] Example templates for all Custom Resource Definitions intended to be used
* [x] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://sdk.operatorframework.io/docs/overview/#operator-capability-level)

### CODE REVIEW

**Comment by haibinxie** [comment]
@J0zi @mvalarh could you see if this is the pipeline issue, looks like opm command argument is incorrect. 

fatal: [localhost]: FAILED! => changed=true 
  cmd: '/tmp/operator-test/bin/opm alpha bundle generate --directory 1.0.3 -u /home/runner/work/community-operators/community-operators/operators/operators/portworx --package portworx '
  delta: '0:00:00.292678'
  end: '2022-12-06 02:44:26.595763'
  msg: non-zero return code
  rc: 1
  start: '2022-12-06 02:44:26.303085'
  stderr: |-
    time="2022-12-06T02:44:26Z" level=info msg="Bundle channels information not provided, inferring from parent package directory"
    Error: please manually input channels and packageName, error interpreting bundle from directory /home/runner/work/community-operators/community-operators/operators/operators/portworx/1.0.3, error parsing package.yaml file in the package root directory, valid PackageManifest YAML file not found in /home/runner/work/community-operators/community-operators/operators/operators/portworx, [error parsing /home/runner/work/community-operators/community-operators/operators/operators/portworx/ci.yaml as package.yaml file, name of package (packageName) is missing, error parsing /home/runner/work/community-operators/community-operators/operators/operators/portworx/portworx.package.yaml as package.yaml file, could not decode contents into package manifest - error unmarshaling JSON: while decoding JSON: json: cannot unmarshal object into Go struct field PackageManifest.channels of type []registry.PackageChannel]
    Usage:
      opm alpha bundle generate [flags]
  
    Flags:
      -c, --channels directory   The list of channels that bundle image belongs to(Required if directory is not pointing to a bundle in the nested bundle format)
      -e, --default string       The default channel for the bundle image
      -d, --directory string     The directory where bundle manifests for a specific version are located.
      -h, --help                 help for generate
      -u, --output-dir string    Optional output directory for operator manifests
      -p, --package directory    The name of the package that bundle image belongs to (Required if directory is not pointing to a bundle in the nested bundle format)
  
    Global Flags:
          --skip-tls-verify   skip TLS certificate verification for container image registries while pulling bundles
          --use-http          use plain HTTP for container image registries while pulling bundles
  stderr_lines: <omitted>
  stdout: ''
  stdout_lines: <omitted>

PLAY RECAP *********************************************************************
localhost                  : ok=109  changed=17   unreachable=0    failed=1    skipped=62   rescued=0    ignored=0

**Review by pure-jliao — APPROVED** [review]

**Comment by haibinxie** [comment]
@J0zi @mvalarh could you take a look at the failure above?

**Comment by J0zi** [comment]
@haibinxie looks like some indentation issue. But cannot find it right way.

File: operators/portworx/portworx.package.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 channels:
-- currentCSV: portworx-operator.v1.10.0
+  currentCSV: portworx-operator.v1.10.1
   name: alpha
-- currentCSV: portworx-operator.v1.10.0
```
Comment (mvala) [inline operators/portworx/portworx.package.yaml:4]: `-` should stay as it is element of array in `channels`

**Pull request merged**